JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,15 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  const hours = Math.floor(seconds / 3600);
+  const minutes = Math.floor((seconds % 3600) / 60);
+  const remainingSeconds = seconds % 60;
+
+  const units = [hours, minutes, remainingSeconds];
+
+  const formattedUnits = units.map((unit) => unit.toString().padStart(2, '0'));
```

Comment:
Nice way yo apply the needed unit transformation, I would prefer use timeUnits, instead of units but good work 👍